### ISSUE TITLE
Incorrect typing when following recipes example

### ISSUE THREAD

**Issue by sushantdhiman** [Norserium/react-advanced-cropper#2]
Recent update to types in `advanced-cropper@0.5.0`

https://github.com/Norserium/advanced-cropper/commit/9e1f9d0b2acedb3ec928420ffae8c134404eaca3#diff-548739e5d5fd25933664a482d08d17882450cb3a3401c3c3c0f3f7c8f390bca1R454

```diff
- export function getMimeType(arrayBuffer, fallback = null) {
+ export function getMimeType(arrayBuffer: ArrayBuffer, fallback = null) {
```

Gives typescript compilation error when using code from `recipes#load-image-from-a-disc`

- https://github.com/Norserium/react-advanced-cropper/blob/master/example/docs/guides/recipes.mdx
- https://norserium.github.io/react-advanced-cropper/docs/guides/recipes#load-image-from-a-disc

```ts
Argument of type 'string | ArrayBuffer | null | undefined' is not assignable to parameter of type 'ArrayBuffer'.
  Type 'undefined' is not assignable to type 'ArrayBuffer'.

59           type: getMimeType(e.target?.result, typeFallback),
```

I think `getMimeType` should accept `string` or even `null`(so it can fallback to 2nd argument)

**Comment by sushantdhiman** [comment]
Additionally 2nd argument should accept `string`. Overall I think correct typing should be

```ts
export function getMimeType(arrayBuffer: ArrayBuffer | string | null, fallback: string) {
```

**Comment by Norserium** [comment]
@sushantdhiman, thank you for such detailed report.

I am currently working on the next version. It will fix this issue. Also I will reexamine all the examples to ensure, that all will be okay now. However if this process drags on, I will publish a immediate release that fixes this specific issue.

Stay in touch.

**Comment by Norserium** [comment]
@sushantdhiman, fixed in `0.6.2`. The next big release is delayed.
